packages/neuron-wallet/src/services/transaction-sender.ts

```diff
@@ -417,8 +417,9 @@ export default class TransactionSender {
     const message = blake2b.digest()
 
     if (!wallet.isHardware()) {
-      const keyPair = new ECPair(privateKeyOrPath)
-      emptyWitness.lock = keyPair.signRecoverable(message)
+      // `privateKeyOrPath` variable here is path only when wallet is hardware, otherwise it's a private key
```

Comment:
```suggestion
      // `privateKeyOrPath` variable here is a private key because wallet is not a hardware one. Otherwise, it will be a private key path.
```

Reorder the expression because it's in the branch of `(!isHardware())`